### PR TITLE
chore: bump to latest version

### PR DESCRIPTION
**Pull request by sarahschwartz**
This PR bumps the docs to reflect the latest version of fuel-core

### CODE REVIEW

File: docs/reference/objects.mdx [hunk 1]
```diff
@@ -836,6 +957,10 @@ The binary Merkle tree root of state, balances, and the contracts code hash of e
 
 The binary merkle tree root of all genesis messages.
 
+`transactionsRoot`: [`Bytes32!`](/docs/reference/scalars/#bytes32)
+
+The binary merkle tree root of all previous transactions.
```
Comment (bvrooman) [inline docs/reference/objects.mdx]:
```suggestion
The binary Merkle tree root of all previous transactions.
```

File: docs/reference/objects.mdx [hunk 2]
```diff
@@ -876,9 +1009,13 @@ The number of receipt messages in the block.
 
 The merkle root of the transactions in the block.
 
-`messageReceiptRoot`: [`Bytes32!`](/docs/reference/scalars/#bytes32)
+`messageOutboxRoot`: [`Bytes32!`](/docs/reference/scalars/#bytes32)
+
+The merkle root of the outgoing messages back to the data availability (DA) layer from Fuel, where the inputs are the IDs (`MessageId`) of the messages. The IDs are produced by executing transactions and collecting IDs from the receipts of the `Message` outputs.
```
Comment (bvrooman) [inline docs/reference/objects.mdx]:
```suggestion
The Merkle root of the outgoing messages back to the data availability (DA) layer from Fuel, where the inputs are the IDs (`MessageId`) of the messages. The IDs are produced by executing transactions and collecting IDs from the receipts of the `Message` outputs.
```

File: docs/reference/objects.mdx [hunk 3]
```diff
@@ -876,9 +1009,13 @@ The number of receipt messages in the block.
...
 
-The merkle root of the receipt messages in the block.
+`eventInboxRoot`: [`Bytes32!`](/docs/reference/scalars/#bytes32)
+
+The merkle root of the incoming events from the data availability (DA) layer to Fuel, where the tree inputs are the hashes of these events.
```
Comment (bvrooman) [inline docs/reference/objects.mdx]:
```suggestion
The Merkle root of the incoming events from the data availability (DA) layer to Fuel, where the tree inputs are the hashes of these events.
```

File: docs/reference/objects.mdx [hunk 4]
```diff
@@ -1624,6 +1807,26 @@ The salt value for the transaction.
 
 An array of storage slot.
 
+`bytecodeRoot`: [`Bytes32`](/docs/reference/scalars/#bytes32)
+
+The merkle tree root of the bytecode that is being uploaded.
```
Comment (bvrooman) [inline docs/reference/objects.mdx]:
```suggestion
The Merkle tree root of the bytecode that is being uploaded.
```

File: docs/reference/objects.mdx [hunk 5]
```diff
@@ -1624,6 +1807,26 @@ The salt value for the transaction.
...
+
+`subsectionIndex`: [`U16`](/docs/reference/scalars/#u16)
+
+The index of the subsection of the bytecode.
+
+`subsectionsNumber`: [`U16`](/docs/reference/scalars/#u16)
+
+The total number of subsections that the bytecode was divided.
```
Comment (bvrooman) [inline docs/reference/objects.mdx]:
```suggestion
The total number of subsections that the bytecode was divided into.
```

**Review by bvrooman — DISMISSED** [review]
I looked over the reference changes and made a few suggestions for spelling (ie. Merkle over merkle, since it's a name). The definitions look correct and straight forward to me. You may want to get additional eyes from others on the Client team if possible. I did not check the correctness of the examples by actually running them - are these changes confirmed? Overall, lgtm.

**Review by Dentosal — COMMENTED** [review]
I agree with @bvrooman here, the doc changes look good, but I didn't try out all of the examples either. Assuming those work, LGTM.

**Review by calldelegation — APPROVED** [review]

**Pull request merged**